components/FavoriteButton.js

```diff
@@ -12,9 +12,19 @@ export default function FavoriteButton({ onToggleFavorite, id, isFavorite }) {
   return (
     <StyledFavButton type="button" onClick={() => handleToggle(id)}>
       {favorite ? (
-        <Image src={"/icons/heart_filled.png"} width={30}  height={30} alt= {"heart icon"}/>
+        <Image
+          src={"/icons/heart_red.png"}
+          width={30}
+          height={30}
+          alt={"heart icon"}
+        />
       ) : (
-        <Image src={"/icons/heart_empty.png"} width={30}  height={30} alt= {"heart icon"}/>
+        <Image
+          src={"/icons/heart_empty.png"}
+          width={30}
+          height={30}
+          alt={"heart icon"}
+        />
       )}
```

Comment:
- There is some redundancy in the `Image `component that could be streamlined by computing the source path based on the `favourite` state.
- the alt attribute could describe the action or state change that will occur if the button is interacted 

_You could manage it like this_ ⬇️

```js
return (
    <StyledFavButton type="button" onClick={() => handleToggle(id)}>
      <Image
        src={favorite ? "/icons/heart_red.png" : "/icons/heart_empty.png"}
        width={30}
        height={30}
        alt={favorite ? "Unfavorite" : "Favorite"}
      />
    </StyledFavButton>
);
```